selinux/ipa.te

```diff
@@ -412,3 +416,28 @@ optional_policy(`
 optional_policy(`
        systemd_private_tmp(ipa_custodia_tmp_t)
 ')
+
+optional_policy(`
+    gen_require(`
+        type tomcat_t;
+    ')
+    can_exec(tomcat_t, ipa_pki_retrieve_key_exec_t)
+    pki_manage_tomcat_etc_rw(ipa_pki_retrieve_key_exec_t)
```

Comment:
```
$ macro-expander 'pki_manage_tomcat_etc_rw(ipa_pki_retrieve_key_exec_t)'
allow ipa_pki_retrieve_key_exec_t pki_tomcat_etc_rw_t:dir { open read getattr lock search ioctl add_name remove_name write };
allow ipa_pki_retrieve_key_exec_t pki_tomcat_etc_rw_t:file { create open getattr setattr read write append rename link unlink ioctl lock };
allow ipa_pki_retrieve_key_exec_t pki_tomcat_etc_rw_t:dir { open read getattr lock search ioctl add_name remove_name write };
allow ipa_pki_retrieve_key_exec_t pki_tomcat_etc_rw_t:lnk_file { create getattr setattr read write append rename link unlink ioctl lock };
```
So here you are granting a process labeled `ipa_pki_retrieve_key_exec_t` to RW some files, but `ipa_pki_retrieve_key_exec_t` is a file label, not a process label.

Reply:
Good catch, that makes sense, thank you.